src/test/webdriver/geckodriver.rs

```diff
@@ -20,8 +21,18 @@ pub fn get_or_install_geckodriver(
     cache: &Cache,
     mode: InstallMode,
 ) -> Result<PathBuf, failure::Error> {
-    if let Ok(path) = which::which("geckodriver") {
-        return Ok(path);
+    // geckodriver Windows binaries >v0.24.0 have an additional
+    // runtime dependency that we cannot be sure is present on the
+    // user's machine
+    //
+    // https://github.com/mozilla/geckodriver/issues/1617
+    //
+    // until this is resolved, always install v0.24.0 on windows
+    if !target::WINDOWS {
```

Comment:
Why don't we use the binary if it already exists?

Reply:
Ah, okay, I would have thought the user would have had lower than 0.26.0, but if it's failing on CI then I agree this is the right approach.